libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -97,6 +97,53 @@ void CompilerUtils::revertWithStringData(Type const& _argumentType)
 	m_context << Instruction::REVERT;
 }
 
+void CompilerUtils::accessCalldataTail(Type const& _type)
```

Comment:
Any change in this function?

Reply:
Shouldn't be changed apart from some renaming.